homeassistant/components/ecowitt/__init__.py

```diff
@@ -33,6 +33,15 @@ async def handle_webhook(
         hass, DOMAIN, entry.title, entry.data[CONF_WEBHOOK_ID], handle_webhook
     )
 
+    def async_update_device_config(sensor: EcoWittSensor) -> None:
+        """Update device config."""
+        hass.config_entries.async_update_entry(
+            entry,
+            data={**entry.data, "station": sensor.station},
```

Comment:
Is it possible to have multiple devices on the same webhook? If yes this solution will not work